resources/views/Components/FeedbackComment/FeedbackComment_component.twig

```diff
@@ -4,7 +4,7 @@
 
             template: '#vue-feedback-comment',
             delimiters: ["${", "}"],
-            props: ['feedback', 'itemAttributes', 'authenticatedUser', 'isReply', 'showControls'],
+            props: ['feedback', 'itemAttributes', 'authenticatedUser', 'isReply', 'showControls', 'paddingClasses', 'paddingInlineStyles', 'options'],
```

Comment:
Property `options` kürzen. Es wird nur `options.timestampVisibility.mobile` verwendet !

Reply:
Da ein Comment ein ListEntry als Child-Component haben kann, muss auch `options.releaseFeedbacks` in dem OptionsSet bleiben.

Reply:
done